test/client_test.py

```diff
@@ -151,3 +151,44 @@ def test_client_from_env(servicer):
         assert client_4 == client_3
     finally:
         Client.set_env_client(None)
+
+
+def test_multiple_profile_error(servicer, modal_config):
+    config = """
```

Comment:
Love the structure of these tests after your addition of the `modal_config` ctx manager :)